front-end/app/global.scss

```diff
@@ -33,42 +33,46 @@
   }
 }
 
-
 .footer-primary {
-  div{
-    background: #162E51;
+  div {
+    background: #162e51;
     color: white !important;
     min-height: 84px;
     a {
       color: white !important;
     }
   }
   .usa_footer-trademark {
-    font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif!important;
+    font-family:
+      Source Sans Pro Web,
+      Helvetica Neue,
+      Helvetica,
+      Roboto,
+      Arial,
+      sans-serif !important;
     font-size: 14px;
     font-weight: 300;
     line-height: 23px;
     letter-spacing: 0em;
     text-align: right;
   }
-
 }
 
-// @media all and (min-height: 35rem){
-//   .usa-footer{
-//     position: absolute;
-//     bottom: 0;
-//     width: 100%;
-//   }
-// }
-
+@media all and (min-height: 35rem) {
```

Comment:
We can blame my linter for all the other file changes here...this uncommenting is the only real change in this file!